src/SFA.DAS.EAS.Portal.UnitTests/Portal/Application/Adapters/CohortAdapterTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Messages.Events;
+using SFA.DAS.EAS.Portal.Application.Adapters;
+using SFA.DAS.EAS.Portal.UnitTests.Builders;
+
+namespace SFA.DAS.EAS.Portal.UnitTests.Portal.Application.Adapters
+{
+    [Parallelizable]
+    [TestFixture]
+    public class CohortAdapterTests
+    {
+        private readonly CohortAdapter _sut;
+
+        public CohortAdapterTests()
+        {
+            _sut = new CohortAdapter();
+        }
+
+        [Test]
+        public void WhenConvertCalled_ThenTheEventIsMappedToTheCommand()
```

Comment:
This unit test exhibits the 'expected values can be changed by the code under test' issue highlighted in another comment.